db/migrate/20180529102124_add_calculated_value_to_answer_table.rb

```diff
@@ -0,0 +1,7 @@
+# -*- encoding : utf-8 -*-
+
+class AddCalculatedValueToAnswerTable < Card::Migration::DeckStructure
+  def up
+    add_column :answers, :calculated_value, :string
```

Comment:
I know it's a long name, but I'm tempted to call this `overridden_calculated_value`, because it took me a bit to figure out that a calculated answer has no calculated_value unless it's overridden.

if we want it short, I think `overridden_value` is less confusing than `calculated_value`

Reply:
Ugh, I'm sorry that took hours. I don't understand why the output isn't working, but the schema issue is the `without_dumping` thing that I added to migrate.rake to avoid the problem of failures attempting to alter files in the gem.  maybe we need to make it smart about whether the decko code is in a gem or a repo.  I can work on that.